ci/long_running_tests/check_workloads.sh

```diff
@@ -4,21 +4,33 @@
 
 ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
 
-pushd "$ROOT_DIR"
+if [ "$1" == "--load" ]; then
+    check_load=true
```

Comment:
This gives a summary like
```
workload1:  02:23:40 up 53 min,  2 users,  load average: 0.00, 0.02, 0.84
workload2:  02:23:41 up 53 min,  2 users,  load average: 0.00, 0.00, 0.07
workload3:  02:23:43 up 53 min,  2 users,  load average: 0.00, 0.00, 0.22
workload_apex:  02:23:45 up 53 min,  3 users,  load average: 16.85, 7.67, 3.54
workload_impala:  02:23:46 up 53 min,  2 users,  load average: 13.23, 12.72, 11.49
workload_pbt:  02:23:48 up 53 min,  2 users,  load average: 11.04, 11.63, 10.06
```

Which makes it easy to check if everything ok by inspecting if the most recent load is >>1.